data/aliases.ts

```diff
@@ -2355,189 +2521,342 @@ export const Aliases: import('../sim/dex').AliasesTable = {
 	'Jum-pluff',
 	'Kilo-wattrel',
+	'Kin-gambit',
 	'Klin-klang',
 	'Korai-don',
+	'Lea-vanny',
+	'Ludi-colo',
+	'Lumi-neon',
 	'Luv-disc',
+	'Mag-cargo',
 	'Ma-gearna',
-	'Magne-zone',
+	'Magi-karp',
 	'Mag-mortar',
+	'Magne-zone',
 	'Mamo-swine',
+	'Mandi-buzz',
 	'Medi-cham',
+	'Meta-gross',
+	'Mien-foo',
+	'Mien-shao',
 	'Mirai-don',
 	'Mol-tres',
+	'Mud-bray',
+	'Muds-dale',
+	'Munch-lax',
+	'Mur-krow',
+	'Naga-nadel',
 	'Nido-king',
 	'Nido-queen',
 	'Nido-rina',
 	'Nido-rino',
+	'Nihi-lego',
+	'Nine-tales',
+	'Noi-vern',
+	'Nose-pass',
 	'Obsta-goon',
 	'Oger-pon',
+	'Oki-dogi',
+	'Oma-star',
 	'Orth-worm',
 	'Over-qwil',
+	'Palos-sand',
+	'Pan-cham',
+	'Pan-goro',
 	'Pan-pour',
 	'Pan-sage',
 	'Pan-sear',
 	'Pecha-runt',
+	'Perr-serker',
+	'Phero-mosa',
+	'Pilo-swine',
+	'Pinc-urchin',
+	'Poli-toed',
+	'Poli-wrath',
 	'Poltcha-geist',
 	'Poltea-geist',
+	'Prima-rina',
 	'Prime-ape',
 	'Psy-duck',
+	'Pyuku-muku',
+	'Quag-sire',
+	'Quaq-quaval',
+	'Rai-kou',
 	'Reg-ice',
 	'Regi-drago',
 	'Regi-eleki',
 	'Regi-gigas',
 	'Regi-rock',
 	'Regi-steel',
+	'Reli-canth',
+	'Reva-vroom',
 	'Rhy-don',
 	'Ribom-bee',
 	'Rilla-boom',
+	'Rock-ruff',
+	'Rose-rade',
 	'Sala-mence',
+	'Samu-rott',
+	'Sanda-conda',
 	'Sand-shrew',
 	'Sand-slash',
 	'Saws-buck',
+	'Sci-zor',
+	'Scoli-pede',
 	'Sco-villain',
 	'Sea-king',
 	'Seismi-toad',
+	'Sil-vally',
 	'Simi-pour',
 	'Simi-sage',
 	'Simi-sear',
+	'Sizzli-pede',
 	'Skele-dirge',
 	'Slow-bro',
 	'Slow-king',
+	'Snor-lax',
+	'Sno-runt',
+	'Spiri-tomb',
+	'Squawka-billy',
+	'Stak-ataka',
 	'Star-mie',
+	'Ston-journer',
+	'Sudo-woodo',
+	'Sui-cune',
+	'Swam-pert',
 	'Tad-bulb',
 	'Talon-flame',
+	'Tan-growth',
+	'Tatsu-giri',
 	'Tenta-cool',
 	'Tenta-cruel',
+	'Tera-pagos',
+	'Tinka-tink',
+	'Tinka-ton',
+	'Tinka-tuff',
 	'Toeds-cool',
 	'Toeds-cruel',
+	'Toge-kiss',
+	'Tor-terra',
+	'Tou-cannon',
 	'Toxa-pex',
+	'Toxi-croak',
+	'Tox-tricity',
 	'Tyrani-tar',
 	'Ursa-luna',
+	'Ursa-luna Blood-moon',
+	'Ur-shifu',
+	'Veni-pede',
 	'Veno-moth',
 	'Venu-saur',
+	'Vespi-quen',
 	'Vika-volt',
 	'Vile-plume',
+	'Vol-canion',
+	'Vol-carona',
+	'Vul-pix',
+	'Wea-vile',
+	'Whirli-pede',
 	'Wug-trio',
 	'Wyr-deer',
+	'Xurki-tree',
+	'Zama-zenta',
 	'Zap-dos',
 
 	// moves
+	'10,000,000 Volt Thunder-bolt',
 	'Accele-rock',
 	'Acid Down-pour',
 	'Acu-pressure',
+	'Aero-blast',
 	'Aroma-therapy',
+	'Ava-lanche',
 	'Boom-burst',
 	'Bleak-wind Storm',
 	'Break-neck Blitz',
 	'Bull-doze',
+	'Burn-ing Bulwark',
 	'Catastro-pika',
 	'Cease-less Edge',
-	'Crab-hammer',
 	'Chloro-blast',
 	'Clangorous Soul-blaze',
+	'Copy-cat',
 	'Cork-screw Crash',
+	'Crab-hammer',
 	'Dis-charge',
+	'Dual Wing-beat',
 	'Dyna-max Cannon',
 	'Earth-quake',
+	'Electro-web',
 	'Eterna-beam',
 	'Extra-sensory',
 	'Extreme Evo-boost',
 	'Flame-thrower',
+	'Geo-mancy',
 	'Giga-volt Havoc',
 	'Head-butt',
 	'Head-long Rush',
 	'High Horse-power',
+	'Hurri-cane',
 	'Hyper-space Fury',
 	'Hyper-space Hole',
 	'Inferno Over-drive',
+	'Jungle Heal-ing',
```

Comment:
```suggestion
```

I already told you this won't match the things you want it to match. This one only adds `ing`, `jhi`, and `jhing` to the possible matches.